common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3574,6 +3574,12 @@
     ],
     "sqlState" : "42710"
   },
+  "MULTI_ALIAS_WITHOUT_GENERATOR" : {
+    "message" : [
+      "Multi part aliasing (<names>) is not supported with <expr> as it is not a generator function."
```

Comment:
is `Multi part aliasing` an official name in SQL standard?

Reply:
It's not a concept at all. I don't have a better name.